.github/workflows/docs.yml

```diff
@@ -1,5 +1,9 @@
 name: Docs
-on: [push, pull_request, workflow_dispatch]
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ '*' ]
```

Comment:
Changed the workflow so we only build the doc when we push on develop and/or we create a PR.